backend/scripts/test-auth-api.sh

```
@@ -0,0 +1,110 @@
+  echo "❌ 백엔드 서버에 연결할 수 없습니다"
+  echo "   백엔드 서버가 실행 중인지 확인하세요: cd backend && npm run start:dev"
+  exit 1
+fi
+echo ""
+
+# 회원가입 테스트
+echo "2️⃣ 회원가입 테스트..."
+REGISTER_RESPONSE=$(curl -s -X POST "$API_URL/auth/register" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "email": "test_'$(date +%s)'@example.com",
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)